cmd/aterm/appdialogs/first_run_prompt.go

```diff
@@ -0,0 +1,133 @@
+package appdialogs
+
+import (
+	"encoding/json"
+	"io/ioutil"
+
+	"github.com/manifoldco/promptui"
+	"github.com/theparanoids/ashirt-server/backend/dtos"
+	"github.com/theparanoids/aterm/cmd/aterm/config"
+	"github.com/theparanoids/aterm/fancy"
+
+	"github.com/theparanoids/aterm/dialog"
+	"github.com/theparanoids/aterm/errors"
+	"github.com/theparanoids/aterm/network"
+)
+
+// FirstRun collects configuration data when the application is run for the first time. Data can be
+// loaded from an external source (other ASHIRT application). Each question is prefaced with a small
+// description of what is needed
+func FirstRun(primaryConfigFile, pathToCommonConfig string) (config.TermRecorderConfigOverrides, error) {
+	printf("Hi and welcome to the ASHIRT Terminal Recorder. \n"+
+		"\n"+
+		"I couldn't read a configuration file (I looked here: %v). "+
+		"I think this might be the first run of this application. "+
+		"Before we begin recording, we need to configure this application.\n",
+		fancy.WithBold(primaryConfigFile),
+	)
+
+	var configData config.TermRecorderConfigOverrides
+
+	// try to read common config
+	content, err := ioutil.ReadFile(pathToCommonConfig)
+	if err == nil {
+		err = json.Unmarshal(content, &configData)
+		if err == nil {
+			printf("I was able to load defaults from another ASHIRT application. Let's double check these values.\n\n")
+		}
+	}
+
+	printline("If the value in [brackets] looks good, simply press enter to accept that value.")
+	configData.APIURL = askFor(apiURLFields, configData.APIURL).Value
```

Comment:
We probably want to do some sort of canonicalization here. If a user enters a backslash at the end of the api url value here (eg. `http://api.ashirt.com/`) it ends up causing all requests to fail as unauthorized I assume due to the signing logic. We don't need to solve every possible mistake but this seems like it might be a fairly common one.

Reply:
This issue should now be fixed, but I'm not sure what other potential issues we'd fix. If you have a sense for what's in-bounds here, let me know.